src/LineBot/RichMenu/RichMenuBounds.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
+//
+// Dirk Lemstra licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Line
+{
+    /// <summary>
+    /// Object describing the boundaries of the area in pixels.
+    /// </summary>
+    public class RichMenuBounds
+    {
+        private int _x;
+        private int _y;
+        private int _width;
+        private int _height;
+
+        /// <summary>
+        /// Gets or sets the horizontal position relative to the top-left corner of the area.
+        /// </summary>
+        [JsonProperty("x")]
+        public int X
+        {
+            get => _x;
+            set
+            {
+                if (value > 2500)
+                {
+                    throw new InvalidOperationException("The horizontal position cannot be bigger than 2500.");
+                }
+
+                if (value < 0)
+                {
+                    throw new InvalidOperationException("The horizontal position cannot be less than 0.");
+                }
+
+                _x = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the vertical position relative to the top-left corner of the area.
+        /// </summary>
+        [JsonProperty("y")]
+        public int Y
+        {
+            get => _y;
+            set
+            {
+                if (value > 1686)
+                {
+                    throw new InvalidOperationException("The vertical position cannot be bigger than 1686.");
+                }
+
+                if (value < 0)
+                {
+                    throw new InvalidOperationException("The vertical position cannot be less than 0.");
+                }
+
+                _y = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the width of the area.
+        /// </summary>
+        [JsonProperty("width")]
+        public int Width
+        {
+            get => _width;
+            set
+            {
+                if (value > 2500)
```

Comment:
I now wonder if we should write these checks differently. We probably need to check if the new `Width` and `X` would become more than 2500. We probably need a similar check inside the `X` property.

Reply:
Yeah, I considered it but think if users want to set it at the same time, how can we write it then?
Should we add contractor so can check if the value is properly set?

Reply:
If they set it at the same time then one will be set before the other one is set. The properties are not set at exactly the same time.

Reply:
Added check for width and height.